replicatedhq/ship
# Kustomize merged overlays

What I Did
------------
Make sure the production build of React is used when built for production (Resolve #301)

How I Did it
------------
Updated the scripts in the webpack ENV to point to production react builds
Resolve some ESLint errors.

How to verify it
------------
Run ship in production and make sure there is no console error saying the development build is being used in a production environment.

Description for the Changelog
------------
Use production build of React.
Resolve some linter errors


Picture of a Boat (not required but encouraged)
------------
⛵️
